data/korath/korath missions.txt

```diff
@@ -257,3 +257,223 @@ mission "Small Scale Delivery"
+				`	(Abandon the ship)`
+					goto abandon
+			
+			label abandon
+			action
+				fail
+			`	Your crew abandons the Korath ship the elements. One more tragic story amid the ruins of their civilisation.`
+				flee
+			
+			label runforit
+			action
+				# Ship name should match the <EDC-ship> substitution above.
+				give ship "Charm-Shallop (Distressed)" "Esketari"
+			`	You manage to bring up the shields, at least initially, using the <ship>. Your crew on the <EDC-ship> take their stations as you prepare to evade Mereti patrols on your way to <destination>.`
+				launch
+	
+	on complete
+		conversation
+			`There are no crowds waiting for you when you land. Rather the ground crews seem astonished to see the ship. It takes you some time to arrange for repairs but at least they are familiar with the ship. When you come out you see a pavilion has been set up near the <EDC-ship> and small groups of Korath periodically stop by.`
+			choice
+				`	(Check it out.)`
+				`	(I've got places to be.)`
+					decline
+				
+			`	The tent is larger than you expected. At the back there is a group of Korath arranged like a choir, but standing oddly still. A Korath in some kind of robes stands next to a table piled with sweets and cups of some kind of thick drink. As you approach, you realize that the "choir" is made of holograms. You do a quick mental count and guess that these are the Korath that died on the ship; this is probably some type of memorial. The Korath in robes greets you and presses a cup into your hand. It is fermented and has a sweet and sour taste that leaves you lips and mouth tingling and then numb. They press the sweets on you so you take them. You see a few groups from the spaceport follow the same ritual, though some of them walk up to the "choir" and say something to them. When you finish the food, you take your leave and return to your ship.`
+
+
+mission "Efreti: Distress Call 5"
+	name "Translate Title"
+	description "Translate the Korath data chip."
+	source "Kuwaru Efreti"
+	invisible
+	landing
+
+	to offer
+		has "Efreti: Distress Call 4: done"
+	on offer
+		conversation
+			`The Quarg have translated the Korath language for you in the past. Would you like to see if they will tell you what the data chip says?`
+			choice
+				`	(Ask the Quarg to translate the data chip.)`
+				`	(Maybe another time.)`
+					defer
+				`	(I really don't care what it says.)`
+					decline
+			
+			`	You approach one of the Quarg in the spaceport. It stops and acknowledges you but doesn't speak. It pauses for a few moments after you make your request but then holds out its hand. It takes the data chip and slots it into a communicator of some sort, It then nods to itself and hands you back the chip.`
+			`	You plug the chip into your own device and see two files. On opening the second one your read, "Belonging this ship, <EDC-ship>, to alien Human <first> <last>. Grateful from those they helped. Stranded ones thank. Grieving ones thank."`
```

Comment:
This Quarg's got a really terrible translator.

Reply:
```suggestion
			`	You plug the chip into your own device and see two files: one in the Korath language, one in your own. You open the file in your language, and it reads: "Belonging this ship, <EDC-ship>, to alien human <first> <last>. Grateful from those they helped. Stranded ones thank. Grieving ones thank."`
```